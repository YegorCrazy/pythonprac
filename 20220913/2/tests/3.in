[16, 27, 3.55]
