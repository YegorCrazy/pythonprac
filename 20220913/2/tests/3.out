3.55, 16, 27
